data/helpers.d/psql

```diff
@@ -0,0 +1,150 @@
+# Create a master password and set up global settings
+# Please always call this script in install and restore scripts
+#
+# usage: ynh_psql_test_if_first_run
+ynh_psql_test_if_first_run() {
+	if [ -f /etc/yunohost/psql ];
+	then
+		echo "PostgreSQL is already installed, no need to create master password"
+	else
+		pgsql=$(ynh_string_random)
+		pg_hba=""
```

Comment:
What is it ?

Reply:
This is for compatibility between stretch and jessie.

Reply:
Still, what $pg_hba is?
Either a comment should explain what is it, or a concrete variable name.
If someone has to fix anything around here, he should know what's going on and how without trying to decrypt the code.

Reply:
It's used to store the path the the `pg_hba.conf` file ([client authentication control file](https://www.postgresql.org/docs/9.6/static/auth-pg-hba-conf.html)). A comment must be added.